deploy/helm/sumologic/templates/pvc-cleaner/cron-job.yaml

```diff
@@ -0,0 +1,121 @@
+{{- if eq (dig "pvc-cleaner" "logs" "enabled" false (.Values | merge (dict))) true }}
```

Comment:
The original solution provided by Dominik uses some `dig` function in many places, imo it should be replaced by `.Values.pvc-cleaner.*`, but to be honest I have no idea what's the best option.

Reply:
`.Values.pvc-cleaner.*` doesn't work as `-` is not recognizable properly by helm
we can either use `get` and assing `pvc-cleaner` to value or use `dig` as above